scripts/jenkins/run-test.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+set -exuo pipefail
+
+export GOPATH=$WORKSPACE
+export PATH=$PATH:$GOPATH/bin
+
+if [ ! -d "$APM_SERVER_DIR" ] ; then
+  echo "you need to define APM_SERVER_DIR environment variable pointing to the APM server source code"
+  exit 1
+fi
+
+eval "$(gvm ${GO_VERSION})"
+echo "Installing hey-apm dependencies and running unit tests..."
+go get -v -u github.com/golang/dep/cmd/dep
+go get -v -u github.com/graphaelli/hey/requester
```

Comment:
you just need `go get -v -u github.com/golang/dep/cmd/dep` and `dep ensure -v` to install all the rest? or is that what you had problems in the past?

Reply:
IIRC y made C&P from somewhere, I will change it.